rsts/reference/swagger.rst

```diff
@@ -29,4 +29,4 @@ Once sandbox setup is complete, a ready-to-explore message is shown:
    👨‍💻 Flyte is ready! Flyte UI is available at http://localhost:30081/console 🚀 🚀 🎉 
 
 
-Visit http://localhost:30081/openapi to view swagger UI for the service and documentation of the payload fields. To test, send sample queries!
+Visit http://localhost:30081/api/v1/openapi to view swagger UI for the service and documentation of the payload fields. To test, send sample queries!
```

Comment:
@pingsutw http://localhost:30081/api/v1/openapi serves the actual openapi spec as json, not the redoc UI. http://localhost:30081/openapi causes a redirect to http://localhost:8087/openapi/, which is not available without port-forwarding. But if we add a slash at the end (http://localhost:30081/openapi/), it seems to be working.

```suggestion
Visit http://localhost:30081/openapi/ to view swagger UI for the service and documentation of the payload fields. To test, send sample queries!
```